Bots Explained

```diff
@@ -0,0 +1,34 @@
+When my Mentor Oscar Oranagwa asked me to write about bots, I decided to use this opportunity to clarify exactly on what bots
+are, to both my fellow developers as well as any newbie with the desire to benefit from this innovation. I will use a story
+to explain it all to an extent but before that, let me begin with;
+
+WHAT ARE BOTS?
+BOTS are also known as Internet Robots, and just as the name Robot implies; Bots make things easier, they can handle 
+repetitive tasks and many more. Formerly, the popular version of bots came in the form of malware, but that has begun 
+to change with tech companies like Facebook, Telegram, Chatfuel providing a more useful version of this technology, 
```

Comment:
What do think of replacing Chatfuel with say Slack? Since Chatfuel (unlike the others) does not exactly provide a platform to build bot on. Rather it's a platform for building bot for these other platforms.